File: app/src/main/java/com/quoders/apps/madridbus/ui/stopInfo/StopInfoComponent.java
package com.quoders.apps.madridbus.ui.stopInfo;


import com.quoders.apps.madridbus.ApplicationComponent;
import com.quoders.apps.madridbus.di.FragmentScoped;
import com.quoders.apps.madridbus.domain.repository.stopInfo.StopInfoRepositoryModule;

import dagger.Component;

@FragmentScoped
@Component(dependencies = ApplicationComponent.class, modules = {StopInfoPresenterModule.class, StopInfoRepositoryModule.class})
public interface StopInfoComponent {

    void inject(StopInfoActivity stopInfoActivity);
}

File: app/src/main/java/com/quoders/apps/madridbus/ui/stopInfo/StopInfoComponent.kt
package com.quoders.apps.madridbus.ui.stopInfo


import com.quoders.apps.madridbus.ApplicationComponent
import com.quoders.apps.madridbus.di.FragmentScoped

import dagger.Component

@FragmentScoped
@Component(dependencies = arrayOf(ApplicationComponent::class), modules = arrayOf(StopInfoPresenterModule::class))
interface StopInfoComponent {

    fun inject(stopInfoActivity: StopInfoActivity)
}
